.github/workflows/upm-subtree-split.yml

```diff
@@ -0,0 +1,31 @@
+#Action created by Stephen Hodgson
+#Check out https://github.com/RageAgainstThePixel/upm-template !!!
+name: upm-subtree-split
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  upm-subtree-split:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+    - name: upm subtree split
+      run: |
```

Comment:
I have an action for this.

Reply:
https://github.com/RageAgainstThePixel/upm-subtree-split

Reply:
Think I've added it correctly, now?